requirements.txt

```
@@ -1,5 +1,5 @@
 yapf==0.27.0
-mypy==0.711
+mypy>=0.711
```

I'd prefer for `requirements.txt` to actually be a snapshot of known-working dependencies. This range leaves open the possibility that it may break again the future.

How about something like, e.g.:

```suggestion
mypy>=0.711, <=0.761
```

---

Alright, done :)